content/chat/setup.textile

```diff
@@ -66,6 +66,29 @@ import ChatClient from '@ably/chat';
 import { ChatClientProvider } from '@ably/chat/react';
 ```
 
+blang[react].
+  h3(#react-native). React Native
+
+  If you're using React Native, you may need to perform extra steps to use the Chat SDK.
```

Comment:
"May need to perform extra steps", or "will need to"?

Reply:
This is a "may" - it depends on how their React Native project has been configured :)